app/assets/stylesheets/items.scss

```diff
@@ -0,0 +1,59 @@
+.jumbotron {
+  background-color: #94618E !important;
```

Comment:
If you need to use important, there's usually a better way to accomplish what you need.